bundle/compliance/policy/audit/file/ensure_permissions.rego

```diff
@@ -0,0 +1,22 @@
+package compliance.policy.audit.file.ensure_permissions
+
+import data.compliance.lib.common
+import data.compliance.policy.data_adapters.file as data_adapter
+
+finding(user, group, other) := result {
+	mode := data_adapter.filemode
+	rule_evaluation := common.file_permission_match(mode, user, group, other)
+
+	# set result
+	result := {
+		"evaluation": common.calculate_result(rule_evaluation),
+		"expected": {"filemode": ((user * 100) + (group * 10)) + other},
+		"evidence": {"filemode": mode},
```

Comment:
I think the result building can be extracted to a common function.
having to write the three literal strings "evaluation" "expected" "evidence" in every audit is kind of fragile and hard to refactor